cookbook/integrations/aws/athena/athena.py

```diff
@@ -0,0 +1,84 @@
+from flytekit import dynamic, kwtypes, task, workflow
+from flytekit.types.file import FlyteFile
+from flytekit.types.schema import FlyteSchema
+from flytekitplugins.athena import AthenaConfig, AthenaTask
+
+# %%
+# This is the world's simplest query. Note that in order for registration to work properly, you'll need to give your
+# Athena task a name that's unique across your project/domain for your Flyte installation.
+athena_task_no_io = AthenaTask(
+    name="sql.athena.no_io",
+    inputs={},
+    query_template="""
+        select 1
+    """,
+    output_schema_type=None,
+    task_config=AthenaConfig(database="mnist"),
+)
+
+
+@workflow
+def no_io_wf():
+    return athena_task_no_io()
+
+
+# %%
+# Of course, in real world applications we are usually more interested in using Athena to query a dataset.
+# In this case we assume that the vaccinations table exists and has been populated data according to this schema
+#     +----------------------------------------------+
+#     | country (string)                             |
+#     +----------------------------------------------+
+#     | iso_code (string)                            |
+#     +----------------------------------------------+
+#     | date (string)                                |
+#     +----------------------------------------------+
+#     | total_vaccinations (string)                  |
+#     +----------------------------------------------+
+#     | people_vaccinated (string)                   |
+#     +----------------------------------------------+
+#     | people_fully_vaccinated (string)             |
+#     +----------------------------------------------+
+#     | daily_vaccinations_raw (string)              |
+#     +----------------------------------------------+
+#     | daily_vaccinations (string)                  |
+#     +----------------------------------------------+
+#     | total_vaccinations_per_hundred (string)      |
+#     +----------------------------------------------+
+#     | people_vaccinated_per_hundred (string)       |
+#     +----------------------------------------------+
+#     | people_fully_vaccinated_per_hundred (string) |
+#     +----------------------------------------------+
+#     | daily_vaccinations_per_million (string)      |
+#     +----------------------------------------------+
+#     | vaccines (string)                            |
+#     +----------------------------------------------+
+#     | source_name (string)                         |
+#     +----------------------------------------------+
+#     | source_website (string)                      |
+#     +----------------------------------------------+
+#
+# Let's look out how we can parameterize our query to filter results for a specific country.
```

Comment:
I think you might want to explain the steps to actually upload and create the dataset in Athena... or at the very least find an AWS doc on how to do that...

Reply:
good point, done